src/cattrs/converters.py

```diff
@@ -713,6 +713,16 @@ def _structure_simple_literal(val, type):
             raise Exception(f"{val} not in literal {type}")
         return val
 
+    def _structure_enum(self, val: Any, cl: type[Enum]) -> Enum:
+        """Structure ``val`` if possible and return the enum it corresponds to.
+
+        Uses type hints for the "_value_" attribute if they exist to structure
+        the enum values before returning the result."""
+        hints = get_type_hints(cl)
```

Comment:
`get_type_hints` is probably going to be expensive. Let's try doing a simple check first. Maybe something like:

```
if "_value_" in cl.__dict__:
    val = self.structure(val, get_type_hints(cl)["_value_"])
```
I want to avoid doing too much extra work in the majority of cases.